src/renderer/viz/expressions/aggregation/cluster/ClusterCount.js

```diff
@@ -0,0 +1,52 @@
+import BaseExpression from '../../base';
+import { checkMaxArguments } from '../../utils';
+import { CLUSTER_FEATURE_COUNT } from '../../../../../client/windshaft';
+
+/**
+ * Count of features per cluster.
+ *
+ * Note: `clusterCount` has no input parameters and if data is not aggregated, it always returns 1
+ *
+ * @return {Number} Cluster feature count
+ *
+ * @example <caption>Use cluster count as width.</caption>
+ * const s = carto.expressions;
+ * const viz = new carto.Viz({
+ *   width: s.clusterCount() / 50
+ * });
+ *
+ * @example <caption>Use cluster cluster count as width. (String)</caption>
```

Comment:
`cluster` is repeated twice. String examples should always match JS examples with the addition of ` (String)` since the dev-center uses that to put the String examples and the JS ones in two columns